README.md

```diff
@@ -86,6 +86,17 @@ for each table that includes a `resource_owner_id` column:
 add_foreign_key :table_name, :users, column: :resource_owner_id
 ```
 
+Remember to add associations to your model so the related records are deleted.
+If you don't do this a `ActiveRecord::InvalidForeignKey`-error will be raised
+when you try to delete a model with related access grants or access tokens.
```

Comment:
`delete` method will ignore callbacks for associations, so let's write "_try to **`destroy`** the model_" instead to avoid misunderstanding.